tests/boulder-integration.sh

```diff
@@ -44,6 +44,22 @@ common --domains le3.wtf install \
        --cert-path "${root}/csr/cert.pem" \
        --key-path "${root}/csr/key.pem"
 
+# This won't renew (because it's not time yet)
+common renew
+
+# This will renew
+sed -i "4arenew_before_expiry = 10 years" "$root/conf/renewal/le1.wtf.conf"
+common renew
```

Comment:
I think we should do it before 0.4.0 (perhaps in a separate PR).  It should be easy: just count the files in the `archive/domain/` config subdirectory.

We should also verify that some configuration state has persisted across these runs via the renewal config file, and/but that supply arguments to a renewal run causes them to be respected.